ipyvolume/pylab.py

```diff
@@ -389,20 +392,26 @@ def scatter(x, y, z, color=default_color, size=default_size, size_selected=defau
     :param size: {size}
     :param size_selected: like size, but for selected glyphs
     :param color_selected:  like color, but for selected glyphs
-    :param marker: {marker}
+    :param marker: Uses marker
```

Comment:
Why replace this docstring (which is substituted by the snippets on top) by this?

Reply:
sorry that's part of an older segment of changes - will revert that

Reply:
:+1: